cfgov/v1/jinja2/v1/includes/molecules/info-unit-2.html

```diff
@@ -122,7 +124,11 @@
                             href="{{ link.url }}"
                             {%- if link.aria_label -%}
                                 aria-label="{{ link.aria_label }}"
-                            {%- endif %}>{{ link_text }}</a>
+                            {%- endif %}>
+                            {% if link.make_link_boldface %}<strong>{% endif %}
+                            {{ link_text }}
+                            {% if link.make_link_boldface %}</strong>{% endif %}
```

Comment:
![Pakled saying "We are strong now."](https://github.com/cfpb/consumerfinance.gov/assets/10562538/f929db03-5cd2-445e-b9c0-13885a3fefd7)